src/Monsters/Races/Monsters/Manticores.js

```diff
@@ -0,0 +1,50 @@
+import Monsters from "../Monsters";
+import {AmbushMagicModifier} from "../../SpecialRules/AmbushMagic";
+import Fly from "../../SpecialRules/Fly";
+import Poison from "../../SpecialRules/Poison";
+import Terror from "../../SpecialRules/Terror";
+
+const description = "The Manticore is a gigantic lion-like monster with large leathery wings and a spiked tail. They " +
+    "are devastating fighters, attacking with raking claws and long, sharp teeth.";
+
+const manticoreSting = {
+    name: "Manticore Sting",
+    description: `
+        <p>
+            During the Monsters' Phase, in addition to making its normal attacks and after it has moved, a Manticore may
+            make a special poisonous Sting attack. Each Warrior adjacent to the Manticore is automatically hit and
+            suffers 3D6 Wounds, modified for armour and Toughness. As this attack is poisonous. all the rules for the
```

Comment:
Should you replace the full stop after poisonous with a comma? (Not really important to the code, I'm just being picky about the grammar)